app/views/developers/_form.html.erb

```diff
@@ -40,9 +40,9 @@
             </div>
 
             <div class="sm:col-span-6">
-              <%= form.label :time_zone, class: "block text-sm font-medium text-gray-700" %>
+              <%= form.label :primary_time_zone, class: "block text-sm font-medium text-gray-700" %>
               <div class="mt-1">
-                <%= form.time_zone_select(:time_zone, ActiveSupport::TimeZone.us_zones, {include_blank: true}, {class: field_classes(form, :time_zone)}) %>
+                <%= form.primary_time_zone_select(:primary_time_zone, ActiveSupport::TimeZone.us_zones, {include_blank: true}, {class: field_classes(form, :primary_time_zone)}) %>
```

Comment:
I think this is the issue. `#primary_time_zone_select` isn't a method on the form builder.